packages/config/src/mutations/update.js

```diff
@@ -94,15 +96,19 @@ const copyIfExists = async function (src, dest) {
   await cpFile(src, dest)
 }
 
+const deleteNoError = async (path) => {
+  try {
+    await pUnlink(path)
```

Comment:
This saves a call to `pathExists`, but might suppress errors.
I could go either way here

Reply:
This works.

My only concern is whether this might hide the intent behind this call inside `copyOrDelete()`: if `{tempDir}/netlify.toml` does not exist, but `configPath` does, then it means `configPath` was created by `updateConfig()`, therefore should be deleted by `backupConfig()`.

In effect, `deleteNoError()` works the same way though, so this is good.